de.metas.contracts/src/main/java/de/metas/contracts/flatrate/process/C_Flatrate_Term_ChangePriceQty.java

```diff
@@ -0,0 +1,35 @@
+package de.metas.contracts.flatrate.process;
+
+import java.math.BigDecimal;
+
+import org.compiere.Adempiere;
+
+import de.metas.contracts.impl.ContractChangePriceQtyRepository;
+import de.metas.contracts.model.I_C_Flatrate_Term;
+import de.metas.process.JavaProcess;
+import de.metas.process.Param;
+
+public class C_Flatrate_Term_ChangePriceQty extends JavaProcess
+{
+
+	public static final String PARAM_PRICEACTUAL = I_C_Flatrate_Term.COLUMNNAME_PriceActual;
+	public static final String PARAM_PLANNED_QTY_PER_UNIT = I_C_Flatrate_Term.COLUMNNAME_PlannedQtyPerUnit;
+
+
+	@Param(parameterName = PARAM_PRICEACTUAL, mandatory = false)
```

Comment:
i would suggest to just inline the column names here, and not have the two constants..as far as i ca see therey aren't used anywhere outside this process class.
....or ...teo sais, it's ok to have them, but make them `private`

Reply:
agree